.travis.yml

```diff
@@ -13,3 +13,9 @@ notifications:
     - irc.mozilla.org#amo-bots
     on_success: change
     on_failure: always
+deploy:
+  # Kumar McMillan's npm API key:
+  api_key:
+    secure: YSa7NJ0M1T5uR8AQdcqMOOcjOcmRbX3ppPa6Rxv+xtqAJn8aTDTVulidXWX1O4f+z1VejF7gzSz8ZsbhRtDNJdaWR7t2nyZiNq5W8qSA9ag467DGdGKIEdG6MXQHetmQi6Qu23k5BpySNlhZUYCvxg8B+XEYRGw2eP48YeraJMPNZkR60pj89x7nYWcz5KIm+h1F4H3PsBROt/ZkERMuNslKtg0K5FgSQ1HJ6++O/blZJ+nhslY3CKDZNX1vV6UAiOyeeP1mQFxskAsQ0Njv6dYQaSrCr/Nc95eoO0khHQ5TPxLydjXhaCFcif7sDK2/8LE94PUEhH8Zl2vlvGuYnFVOmmiqqKRVM4RCI/XwMutUui6d8eq4fp3rUpI8dlF6yvAB2W0Qf+KPXMkw/OU8Kqp8KmrGmGPFTa4vlLto+nE6DKXCbzBtXrmZF5+uReh6ECmXCAg5Spm75dqkCWdAEuhFv8ZkYgDOTg6r7izG6zSQ+DVh6gsqWJEkeIEGmcPx5q16JNVhzCVO/yPjimFh9zSMHjMJ+pYmcAVvUYR3c0AFlZto/jpsZfx7P0G78YEGXT8vFg5N2amBwil+WWIAyUmjHiSjadppMyGh8UVsXLE7c05VkzSGafJ/Tor9MB8dh5497jYtfk4rJ9F65cNkrHvR/5QPsYfVupazXvwLCxI=
+  on:
+    tags: true
```

Comment:
There's an add-ons robot we can use, i think @muffinresearch has emailed us about this. 

https://www.npmjs.com/~addons-robot

With a picture like that, how could you resist?